src/main/java/com/scottlogic/librarygradproject/BookRepository.java

```diff
@@ -75,4 +75,11 @@ public void update(Book entity, int id) {
             throw new BookNotFoundException(id);
         }
     }
+
+    public void removeMultiple(List<Integer> ids) {
+        ids.forEach(id -> {
+            Book bookToRemove = get(id);
```

Comment:
Do we want to do some sort of check to see whether the book exists before we try to remove it?

Reply:
perfect